docs/learning-the-basics.md

```diff
@@ -6,27 +6,27 @@ As with every operating system there are some things that you may not be familia
 The elementary OS desktop is very simple and easy to learn. It consists of two elements: the panel and the dock. You can customize the desktop's wallpaper through **[System Settings](/docs/learning-the-basics#system-settings)** → **Desktop** → **Wallpaper**.
 
 ### The Panel {#the-panel}
-At the top of the screen you can see the Panel. On the left is the Applications menu, in the center are the time and date, and on the right are the Indicators.
+At the top of the screen you can see the Panel. On the left is the Applications Menu, in the center are the time and date, and on the right are the Indicators.
 
 ![The Panel](../images/docs/learning-the-basics/panel.png)
 
-#### Applications {#applications}
+#### Applications Menu {#applications}
 
-On the left side of the panel is the Applications item. Clicking **Applications** brings up a launcher with all of your installed apps. You can view multiple pages of apps using the pagers at the bottom or by scrolling. You can also use the view switcher at the top to switch between a grid view and a category view.
+On the left side of the panel is the Applications Menu. Selecting **Applications** opens the menu with all of your installed apps. You can view multiple pages of apps using the pagers at the bottom or by scrolling. You can also use the view switcher at the top to switch between a grid view and a category view.
```

Comment:
Kind of a special case here. We're referring to the UI, which is why we bold it, so when we refer to "menu" it's not a proper noun. But we use that word for consistency instead of "launcher."